src/main/scala/fi/vm/sade/omatsivut/security/fake/FakeShibbolethServlet.scala

```diff
@@ -36,7 +37,15 @@ class FakeShibbolethServlet(val appConfig: AppConfig) extends OmatSivutServletBa
       }
       case _ => {
         val service = new RemoteAuthenticationInfoService(appConfig.settings.authenticationServiceConfig, appConfig.settings.securitySettings)
-        service.getHenkiloOID(hetu)
+        try {
+          service.getHenkiloOID(hetu)
+        }
+        catch {
+          case e: ParseException => {
+            logger.error(e.failure.details)
+            throw e
+          }
+        }
```

Comment:
Pikkudetailjina minä olisin käyttänyt Scalan Try-rakennetta tässä enkä suoraan try/catchia, mutta jos codebasessa on muuallakin tehty näin, tämä kelpaa oikein hyvin.